app/Main.hs

```diff
@@ -30,21 +44,34 @@ main = execParser opts >>= dotEnv
      <> header "dotenv - loads options from dotenv files" )
 
 config :: Parser Options
-config = Options
-     <$> some (strOption (
-                  long "file"
-                  <> short 'f'
-                  <> metavar "FILE"
-                  <> help "File to read for options" ))
+config =
+  Options
+     <$> argument str (metavar "PROGRAM")
+
+     <*> strOption
+          ( long "env"
+          <> value ".env"
+          <> metavar "\"DOTENV FILE\""
+          <> help "File with the env variables" )
 
-     <*> argument str (metavar "PROGRAM")
+     <*> strOption
+          ( long "env-example"
+          <> value ".env.example"
+          <> metavar "\"DOTENV EXAMPLE\" FILE"
+          <> help "File with all the necesary env variables" )
 
-     <*> switch ( long "overload"
-                  <> short 'o'
-                  <> help "Specify this flag to override existing variables" )
+     <*> switch
+           ( long "overload"
+           <> short 'o'
+           <> help "Specify this flag to override existing variables" )
 
 dotEnv :: MonadIO m => Options -> m ()
-dotEnv opts = liftIO $ do
-  mapM_ (loadFile (overload opts)) (files opts)
-  code <- system (program opts)
+dotEnv Options{..} = liftIO $ do
+  current <- getCurrentDirectory
+
+  let defaultConfig = buildConfig (current </> dotenvFile) (current </> dotenvExampleFile)
```

Comment:
What's the point of prepending `current` to all the paths? If given path is relative, it will be relative to current directory automatically. It should work without `getCurrentDirectory` and `current`, unless I'm missing something.